### ISSUE TITLE
How to add color palette

### ISSUE THREAD

**Issue by Me-Seven-Raccoon** [ealush/emoji-picker-react#321]
Thank you very much for your work. Can you please tell me if there is a way to add a color selection panel like in the demo?
<img width="373" alt="Снимок экрана 2023-01-30 в 15 26 33" src="https://user-images.githubusercontent.com/90495777/215476582-d10b0157-153b-406b-9e12-0f75769d1abf.png">

My plugin
<img width="489" alt="Снимок экрана 2023-01-30 в 15 27 19" src="https://user-images.githubusercontent.com/90495777/215476772-9d1e1887-a97a-49c2-a89b-f03ec77aaea5.png">

**Comment by ealush** [comment]
Long clicking an emoji that supports multiple skin tones (they can be identified by the grey triangle in their corner).
During keyboard navigation, hitting space will pop it open.

**Comment by ealush** [comment]
Yep, that looks like a bug. Let me have a look. Which version do you have installed?

**Comment by ealush** [comment]
I just tested with the latest version (4.4.7) and it seems to be functioning correctly on my end. https://codesandbox.io/s/emoji-picker-react-4-forked-9j7vv3 Can you please share a repro?
